src/Akka.Persistence.Sql.Linq2Db/Journal/Linq2DbWriteJournal.cs

```diff
@@ -47,17 +47,21 @@ public static DateTime FromUnixEpochMillis(in long unixEpochMillis)
 
     public class Linq2DbWriteJournal : AsyncWriteJournal
     {
+        [Obsolete(message: "Use Linq2DbPersistence.Get(ActorSystem).DefaultConfig instead")]
         public static Configuration.Config DefaultConfiguration =>
```

Comment:
Change this to `readonly` field rather than a property?

Reply:
Good idea

Reply:
Done